CMakeLists.txt

```diff
@@ -945,13 +946,6 @@ include_directories("${PROJECT_BINARY_DIR}/src/")
 
 ###############################################################################
 
-###############################################################################
-# RPATH settings
-# Conan set this to true, which is not the cmake default and not what we want
-# for the openstudio targets
-set(CMAKE_SKIP_RPATH FALSE)
```

Comment:
This wasn't working. It's because conan CACHES it with a `1`, and I think cmake just ignores us because of the different type.

Reply:
in conanbuildinfo.cmake, when KEEP_RPATHS isn't passed: 

```cmake
set(CMAKE_SKIP_RPATH 1 CACHE BOOL "rpaths" FORCE)
```